zk.js/src/wallet/user.ts

```diff
@@ -1018,7 +1018,13 @@ export class User {
 
     let relayerMerkleTreeUpdateResponse = "success";
 
-    if (process.env.LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS != "true") {
+    if (
+      process.env.LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS == "false" ||
+      process.env.NEXT_PUBLIC_LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS == "false" ||
+      process.env.REACT_APP_LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS == "false" ||
+      process.env.VUE_APP_LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS == "false" ||
+      process.env.GATSBY_LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS == "false"
+    ) {
```

Comment:
We can extract this check into a function and use it wherever we use  
`if (process.env.LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS != "true")`

https://github.com/Lightprotocol/light-protocol/blob/main/zk.js/src/merkleTree/merkleTreeConfig.ts#L135,
https://github.com/Lightprotocol/light-protocol/blob/main/zk.js/src/relayer.ts#L87, tests, etc

Reply:
changed